.github/workflows/integration-test.yml

```diff
@@ -61,7 +61,7 @@ jobs:
             exit 1
           fi
           export SAST_RESULTS=`jq '.runs | map (.results | length) | add' sast.sarif`
-          expectedSastResults=2
+          expectedSastResults=1
```

Comment:
Actually, `1` is the expected number of SAST finding we need to detect on the test project. I don't know why were the tests passing before.